docs/ecosystem/wash/plugins.md

```diff
@@ -0,0 +1,139 @@
+---
+title: "Wash plugins"
+date: 2024-04-26T11:02:05+06:00
+draft: false
+sidebar_position: 3
+description: "Using wash plugins"
+---
+
+:::tip
+
+Wash plugins are available starting with version 0.28
+
+:::
+
+Wash plugins are a way to extend the functionality of wash. They are Wasm components that can be
+used to add new subcommands to wash. This functionality is experimental, but enabled by default.
+What this means is that we may change the plugin API based on feedback of those using it. However,
+due to the power of the component model, it is very easy to adapt an older version of a plugin to a
+newer version of the API, likely with no changes to the plugin's code and transparent to users.
+
+## Managing and consuming plugins
+
+### Installing a plugin
+
+Installation of a plugin is a relatively straightforward process. A plugin can be installed from a
```

Comment:
Nit: We can probably axe "Installation of a plugin is a relatively straightforward process" and let the process speak for itself.